README.md

```diff
@@ -130,3 +130,32 @@ VITE_ACCRUALS_API_URL_PROXY="http://localhost:9091/"
 ## Troubleshooting
 
 If you’re using an AMD (M1,M2) chip mac, you might encounter problems starting the keycloak docker container. We’ve had some success building a new image locally, as per the instructions in this github thread: https://github.com/docker/for-mac/issues/5310#issuecomment-877653653
+
+## Running UI in LocalDev Environment
+
+### Step 1
+
+Download Callisto LocalDev repository from https://github.com/UKHomeOffice/callisto-localdev
+
+Run its services with command : `docker compose up -d`
+
+### Step 2
+
+Stop (manually) service `web` (most likely the name will be `web-1`)
+
+### Step 3
+
+Create file .env in the root of this project, and add the following:
```

Comment:
If this is needed then we should check in the .env file with these settings and remove it from the git ignore. The aim here is zero config. If localdev is in use, these values wont change. If we are concerned about conflicting with current development not using docker compose (localdev) then we should use a different file name e.g. docker.env and then use the [env_file configuration option](https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option)

Reply:
I guess I will change it to use 2 files:  .env and .env.local
We're going to checkout .env but not .env.local which will be specific for each developer. 
LocalDev variables will be stored in the .env file. This one shouldn't be edited by developers. 
Variables for a solution without using LocalDev will be stored in the .env.local which takes precedence over the .env file. Developers can do anything they want in that file. For example: enabling the usage of mock data etc. 
Is that ok?

Reply:
So this bit won't be needed, we'll check in the .env file with correct settings and remove it from the .gitignore file. I don't know if we need to document the .env.local here or if that's just a common thing to document elsewhere. I presume it's taking advantaged of docker using .env whilst the react app falls back from .env.local to .env allowing us to use it as an override/alternative